test/node/node_test.cpp

```diff
@@ -91,6 +97,30 @@ TEST(NodeTest, MapIteratorWithUndefinedValues) {
   EXPECT_EQ(1, count);
 }
 
+TEST(NodeTest, ConstIteratorOnConstUndefinedNode) {
+  Node node;
+  const Node& cn = node;
+  const Node& undefinedCn = cn["undefined"];
+
+  std::size_t count = 0;
+  for (const_iterator it = undefinedCn.begin(); it != undefinedCn.end(); ++it)
```

Comment:
Add curly braces to this "if" statement. (I haven't changed all existing ones, but new code should put curly braces around all control flow blocks, even one-liners.)